### DISCUSSION TITLE
Login options

### DISCUSSION THREAD

**Discussion by gekomod** [outline/outline#3530] [General] [answered]
hello I have a question I have google in .env and azure but the login options are only slack, do you need to set something else ??

and how to make it possible to browse without logging in?

**Comment by Limezy** [comment, marked as answer]
Hi, have you first tried to login using Slack ? If that's the case, then it's "normal" than only Slack shows.
Indeed, this will be probably fixed soon but for the moment Outline only supports one auth method, and so the first one you use will remain by default. You can always backup all your documents, start a new DB from scratch, use Google or Azure and import back.

Fot your second question, even if it's far from ideal, I made a prototype of a "visitor access" button, where anybody can browse after just clicking on the button. You can find the relevant discussion here : https://github.com/outline/outline/discussions/3143
For this you'll need to build outline from source as it's basically a (monkey) patch
